src/algebra/polynomial/big_operators.lean

```diff
@@ -5,27 +5,24 @@ Authors: Aaron Anderson, Jalex Stark.
 -/
 import data.polynomial.monic
 import tactic.linarith
-
-open polynomial finset
```

Comment:
It's a shame that docstring links don't respect open namespaces

Reply:
Maybe this can be fixed, but atm the main module docstring for the file should come directly after the block of import statements.